src/drafts/hooks/useDynamicTextareaHeight.ts

```diff
@@ -1,12 +1,12 @@
-import {useLayoutEffect, useState} from 'react'
+import {RefObject, useLayoutEffect, useState} from 'react'
 
 import {SxProp} from '../../sx'
 import {getCharacterCoordinates} from '../utils/character-coordinates'
 
 type UseDynamicTextareaHeightSettings = {
-  minHeightLines: number
-  maxHeightLines: number
-  element: HTMLTextAreaElement | null
+  minHeightLines?: number
+  maxHeightLines?: number
```

Comment:
We seem to have default values for these. Would there be any cases where they become undefined here?

Reply:
This hook is exported from drafts, so it's usable outside of the `MarkdownEditor`. In some cases consumers might not want to provide these - I just figured I'd make it more flexible.

Reply:
Ah that makes sense